README.md

```diff
@@ -72,6 +72,7 @@ You can make users **moderators**. They will then be able to **delete any post**
 ## Documentation
 
  * Official documentation: https://git.pleroma.social/pleroma/pleroma/wikis/home
+ * Yunohost apps documentation: https://yunohost.org/#/app_pleroma
```

Comment:
Seems like the Yunohost apps documentation page doesn't work... 
Tested from two browser :/
![image](https://user-images.githubusercontent.com/30271971/65987941-2a207980-e487-11e9-9408-b4923c8ed704.png)

I propose we wait that page to work before merging

Reply:
now it's working